plone/app/discussion/browser/moderation.pt

```diff
@@ -103,8 +104,14 @@
                                     <input type="hidden" name="selected_obj_paths:list" value="#"
                                            tal:attributes="value item/getURL" />
                                 </td>
-                                <td tal:content="python:item.author_name or item.Creator" />
-                                <td tal:content="python:toLocalizedTime(item.ModificationDate, long_format=1)" />
+                                <td>
+                                    <span tal:content="python:item.author_name or item.Creator">Name</span>
+                                    <tal:email tal:condition="email"><br /><a 
```

Comment:
Please, keep each tag on its own line, so it is later on easy to refactor with a minimal diff that will be more easy to read and process, i.e. if someone decides that the ``<br />`` should be replaced by a ``&nbsp;`` or something else that would be a single line change rather removing only the br and adding the nbsp. There will be no way to miss what the change was.